test/index.js

```diff
@@ -787,3 +787,17 @@ describe('decoding (uint[][3], uint)', function () {
     assert.equal(a[0][2][1].toString(10), 6)
   })
 })
+
+//Homebrew tests for simpleEncode() family
```

Comment:
Thanks for adding the test case, great! 👍

(side question: what is a "homebrew test"?? 😄)

Reply:
> what is a "homebrew test"

One we made at home, in contrast with standard or other verified external test vectors such as the official Ethereal ones used in the very first tests.  There are various words for this but Alex Beregszaszi (if `git blame` be believed) already chose 'Homebrew' earlier in the file so I stuck with the convention :-)